src/main/java/frc/robot/subsystems/Shooter.java

```diff
@@ -0,0 +1,107 @@
+package frc.robot.subsystems;
+
+import com.revrobotics.*;
+import edu.wpi.first.math.MathUtil;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.RobotMap;
+
+public class Shooter extends SubsystemBase {
+ private CANSparkMax motor1; //right motor
+ private CANSparkMax motor2; //left motor
+
+ private RelativeEncoder encoder1;
+ private RelativeEncoder encoder2;
+ private SparkPIDController pidController1;
+ private SparkPIDController pidController2;
+ public static double SPEAKER_SPEED_RPM = 4000;
+ public static  double AMP_SPEED_RPM = 2000;
+ public static double TOLERANCE = 150;
+ private static double KF = 0.000185;
+
+ public Shooter(){
+     motor1 = new CANSparkMax(12, CANSparkLowLevel.MotorType.kBrushless);
+     motor2 = new CANSparkMax(14, CANSparkLowLevel.MotorType.kBrushless);
```

Comment:
Use constants in the RobotMap for the motors' IDs

Reply:
I fixed it.